sdk/PrebidServerAdapter/PBServerAdapter.m

```diff
@@ -104,16 +104,8 @@ - (void)requestBidsWithAdUnits:(nullable NSArray<PBAdUnit *> *)adUnits
                         NSString *cacheId = [[NSUUID UUID] UUIDString];
                         NSMutableDictionary *bidCopy = [bid mutableCopy];
                         NSMutableDictionary *adServerTargetingCopy = [bidCopy[@"ext"][@"prebid"][@"targeting"] mutableCopy];
-                        if([adServerTargetingCopy valueForKey:kAPNAdServerCacheIdKey] == nil){
-                            adServerTargetingCopy[kAPNAdServerCacheIdKey] = cacheId;
-                        }
-                        NSMutableDictionary *extCopy = [bidCopy[@"ext"] mutableCopy];
-                        NSMutableDictionary *prebidExtCopy = [bidCopy[@"ext"][@"prebid"] mutableCopy];
-                        prebidExtCopy[@"targeting"] = adServerTargetingCopy;
-                        extCopy[@"prebid"] = prebidExtCopy;
-                        bidCopy[@"ext"] = extCopy;
                         [[PrebidCache globalCache] setObject:bidCopy forKey:cacheId withTimeoutInterval:kAdTimeoutInterval];
```

Comment:
bidobject should be set with the cacheid before setting it in global cache...

Reply:
It doesn't matter right? Setting it after is easier to read.